README.md

```diff
@@ -1,31 +1,17 @@
-> THIS REPOSITORY HAS BEEN ARCHIVED. Open Manufacturing Platform has been formally closed.
+# ESMF documentation site
 
-# sds-documentation
+This repository contains the setup for the documentation of the Eclipse Semantic Modeling Framework
```

Comment:
Proposal: Include link to ESMF project as well.
"This repository contains the setup for the documentation of the Eclipse Semantic Modeling Framework (ESMF, https://projects.eclipse.org/projects/dt.esmf)"